tests/minimal_vf/config.toml

```diff
@@ -0,0 +1,20 @@
+output_file = "MinimalVF.ttf"
+color_format = "glyf_colr_1"
+
+[axis.wght]
+name = "Weight"
+default = 300  # apparently ufo2ft fails if default is not at a master. How silly.
```

Comment:
Agreed the silly comment is itself silly, will remove :D 

Thinking "aloud", it would actually be nice to default to letting the compiler pick the default, and if the user chooses to specify then just do it.

For context, I tend to think if you are programmatically compiling a VF you are into the territory of asserting (falsely or not) that you know what you are doing so it would be reasonable for the compiler to either Just Work when you specify an off-master default (maybe you care about non-VF environments and really need that off-master default) or to fail and clearly indicate how to make it work in the error message (specify some flag if you really want this, beware you may inflate the size of your font perhaps?).